core/src/main/java/org/apache/metamodel/create/AbstractTableCreationBuilder.java

```diff
@@ -123,9 +123,22 @@ public String toSql() {
                     && !column.isNullable().booleanValue()) {
                 sb.append(" NOT NULL");
             }
-            if (column.isPrimaryKey()) {
-                sb.append(" PRIMARY KEY");
-            }
+        }
+        boolean primaryKeyExists = false;
+        for(int i = 0 ; i < columns.length ; i++) {
+            if(columns[i].isPrimaryKey()) {
+                if(!primaryKeyExists) {
+                    sb.append(" ,PRIMARY KEY(");
```

Comment:
Heh, I meant the other space ... like we would normally write (space after each comma, none before):

one, two, three